cli/js/error_stack_test.ts

```diff
@@ -1,8 +1,8 @@
 // Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
 import { test, assert } from "./test_util.ts";
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-const { setPrepareStackTrace } = Deno[Deno.symbols.internal] as any;
+// @ts-ignore
```

Comment:
Was there a specific reason to replace this with `@ts-ignore`?

Reply:
Same issue as https://github.com/denoland/deno/pull/3899#discussion_r377584181